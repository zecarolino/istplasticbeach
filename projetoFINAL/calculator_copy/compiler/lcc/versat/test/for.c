void putchar(int ch) { asm("\trdw R11\n\taddi 2\n\twrw RB\n\trdwb\n\trdwb\n\twrw CPRT_BASE\n"); }
int main() { int i; for (i = 0; i < 7; i++) putchar('0'+i); putchar('\n'); return 0; }
