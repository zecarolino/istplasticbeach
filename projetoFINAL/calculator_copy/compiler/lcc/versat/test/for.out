VCD info: dumpfile xtop.vcd opened for output.
0123456
0
Catched Trap at time  967 clock cycles:
